src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -199,6 +200,34 @@ static inline void cavs_pm_runtime_en_dwdma_clk_gating(uint32_t index)
 #endif
 }
 
+static inline void cavs_pm_core_dis_memory(uint32_t index)
+{
+#if CAVS_VERSION >= CAVS_VERSION_1_8
+	void *core_memory_ptr;
+	extern uintptr_t _sof_core_s_start;
+
+	core_memory_ptr = &_sof_core_s_start + index * SOF_CORE_S_SIZE;
```

Comment:
You are missing first slave core because you are starting with index 1 so the first core_memory_ptr is:
core_memory_ptr = &_sof_core_s_start + 1 * SOF_CORE_S_SIZE;

but actually the first slave core is starting at _sof_core_s_start